.travis.yml

```diff
@@ -6,10 +6,6 @@ env:
 matrix:
   fast_finish: true
   include:
-    - env: DM=~6.0
-      php: 5.3
-    - env: DM=~6.0
```

Comment:
When I worked on this matrix, my idea was to list the PHP versions in increasing order, and DM versions in decreasing order, possibly maximizing compatibility.
```
    - env: DM=@dev
      php: 5.5
    - env: DM=~6.0
      php: 5.6
    - env: DM=~5.0
      php: 7
    - env: DM=~4.2
      php: hhvm
```
What you have now is fine. Thanks.